Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2ButtonCustomization.m

```diff
@@ -0,0 +1,40 @@
+#import "BTThreeDSecureV2BaseCustomization_Internal.h"
+
+#if __has_include(<Braintree/BraintreeThreeDSecure.h>)
+#import <Braintree/BTThreeDSecureV2ButtonCustomization.h>
+#else
+#import <BraintreeThreeDSecure/BTThreeDSecureV2ButtonCustomization.h>
+#endif
+
+// To support SPM without an xcframework version of Cardinal, we created wrappers for Cardinal classes which are substituted for the actual CardinalMobile classes at runtime.
+@implementation BTThreeDSecureV2ButtonCustomization
+
+- (instancetype)init {
+    self = [super init];
+    if (self) {
+        self.cardinalValue = [NSClassFromString(@"ButtonCustomization") new];
+    }
+
+    return self;
+}
+
+- (void)setBackgroundColor:(NSString *)backgroundColor {
```

Comment:
Should there be any validation of the values passed into the color fields to match the format required by Cardinal? (the docs say hex format is required)

Reply:
My initial thought is that we should keep these classes as simple as possible and just pass values along to Cardinal. But I'm curious if anyone else has opinions on this?

Reply:
Also, I just checked to see if Cardinal logs any errors or anything around this, and it doesn't look like they do.

Reply:
Ok then keeping it simple seems reasonable to me, unless anyone else opposes.